tools/accuracy_checker/accuracy_checker/annotation_converters/unicode_character_recognition.py

```diff
@@ -22,8 +22,8 @@
 from ..config import PathField
 
 
-class KondateNakayosiRecognitionDatasetConverter(FileBasedAnnotationConverter):
-    __provider__ = 'kondate_nakayosi_recognition'
+class UnicodeCharacterRecognitionDatasetConverter(FileBasedAnnotationConverter):
+    __provider__ = 'unicode_character_recognition'
```

Comment:
@eaidova Do you think this poses a backwards compatibility issue? Maybe some sort of alias could be added?

Reply:
yes, you are right, I add it after merge these changes